### ISSUE TITLE
Latest version of the Windows GIMP plugin does not work with GIMP 2.10.24

### ISSUE THREAD

**Issue by Wormnest** [c-koi/gmic-qt#107]
Downloaded: https://www.fosshub.com/GMIC.html?dwl=gmic_2.9.6_gimp2.10_win64.exe

Install in GIMP 2.10.24 and it is not working. See GIMP issue [https://gitlab.gnome.org/GNOME/gimp/-/issues/6670](https://gitlab.gnome.org/GNOME/gimp/-/issues/6670).

Looks like it is caused by an incompatible version of libgobject-2.0.0.dll. gmic apparently needs a version that includes function g_memdup2. It should include it's own version of this library instead of relying on the version included with GIMP.

**Comment by dtschump** [comment]
Confirmed. I'm working on it.

**Comment by dtschump** [comment]
Could you test with this installer: https://gmic.eu/files/prerelease/gmic_gimp2.10_win64.exe
(or this zip file : https://gmic.eu/files/prerelease/gmic_gimp2.10_win64.zip).
and tell us if this works ?
Thanks!

**Comment by danilojr** [comment]
It didn't work for me.  I got the same error:  "The procedure entry point g_memdup2 could not be located in the dynamic link library libglib-2.0-0.dll."

**Comment by dtschump** [comment]
Strange. It now works for me, with latest pre-release `2.9.7_pre210401`, on GIMP 2.10.24
(I got the same problem with the previous version of the plug-in, 2.9.6 when I upgraded to GIMP 2.10.24).

![works](https://user-images.githubusercontent.com/10114565/113427187-e63c6680-93d4-11eb-80fd-bd5bd61eda5c.jpg)

Also, I got a report from @paul-andre942 who confirms this also worked for him.

**Comment by danilojr** [comment]
I've tried with the zip file (link above). Are they the same thing (exe and zip)?

**Comment by danilojr** [comment]
Strange...  It's working with the installer.

**Comment by dtschump** [comment]
Yes, the installer and the .zip file contain the same files.

**Comment by killgazmotron** [comment]
> 
> 
> Could you test with this installer: https://gmic.eu/files/prerelease/gmic_gimp2.10_win64.exe
> (or this zip file : https://gmic.eu/files/prerelease/gmic_gimp2.10_win64.zip).
> and tell us if this works ?
> Thanks!

thankyou, this seams to have solved my issue.

**Comment by Stevoh84** [comment]
> Could you test with this installer: https://gmic.eu/files/prerelease/gmic_gimp2.10_win64.exe
> (or this zip file : https://gmic.eu/files/prerelease/gmic_gimp2.10_win64.zip).
> and tell us if this works ?
> Thanks!

.exe has worked for me, thank you

**Comment by pisapiag** [comment]
Tried it today, still doesn't work for me. How will we know when it will work for good? Will we receive an email or do we have to check on a daily basis? Thanks. Keep up the good work.

**Comment by dtschump** [comment]
> Tried it today, still doesn't work for me

Do you have the same error message as before?
The new installer (listed as `latest dev` on the download page) seems to be working for a lot of people.

**Comment by pisapiag** [comment]
Redownloaded it and reinstalled it. It works! Go figure. Thanks dtschump.

**Comment by jamiesizzle** [comment]
Yup, not working for me either. Getting the same error and tried all the offered options with the same result. Will this make Gimp unstable, because every time I start Gimp I get the error. Do you have an uninstall program to get rid of the plugin for now?

**Comment by Wormnest** [comment]
Could it be that when installing a newer version of G'MIC it does not uninstall itself correctly? Meaning that the dll that was causing problems and is not part of 2.9.7 anymore did not get removed.

We had another report [(https://gitlab.gnome.org/GNOME/gimp/-/issues/6771)](https://gitlab.gnome.org/GNOME/gimp/-/issues/6771) where first uninstalling 2.9.6 before installing 2.9.7 fixed the problem.

**Comment by dtschump** [comment]
The issue "The procedure entry point g_memdup2 could not be located in the dynamic link library libglib-2.0-0.dll." has been fixed with latest version 2.9.7. If you still get this message, that means an older version of the plug-in is still installed on your system, and has a higher priority. G'MIC 2.9.7 doesn't use 'g_memdup2' anymore.

Pleas uninstall all previous versions of the G'MIC plug-in before reinstalling 2.9.7. I've seen people having more than 3 installs of G'MIC with different versions!

**Comment by mtedaldi** [comment]
Not sure if it's the same Problem but I've got it with 2.9.8 on windows 10 with Gimp 2.10.14
`the procedure entry point "_ZdaPv" could not be found` (translated from the message in german on my system).

I have no other gmic version installed and tried the installer as well as the zip file. It's always the same message :(

![grafik](https://user-images.githubusercontent.com/1422003/127286335-22bf8b93-52b3-4274-a954-5b0c95c8d3a2.png)
